homeworks_js/h.w-25/javascript.js

```diff
@@ -0,0 +1,205 @@
+который расположен на этой клавише (1–!, 2–@, 3–# и т. д).*/
+let number = 3;
+if (number == 0) {
+  console.log(")")
+} else if (number == 1) {
+  console.log("!")
+} else if (number == 2) {
+  console.log("@")
+} else if (number == 3) {
+  console.log("#") 
+} else if (number == 4) {
+  console.log("$") 
+} else if (number == 5) {
+  console.log("%")
+} else if (number == 6) {
+  console.log("^")
+} else if (number == 7) {
+  console.log("&")
+} else if (number == 8) {
+  console.log("*")
+} else if (number == 9) {
+  console.log("(")
+} else {
+  console.log("wrong")
+}
+
+//3. Запросить у пользователя трехзначное число и проверить, есть ли в нем одинаковые цифры.
+let number = 155;
+let number1 = parseInt(number / 100);
+let number2 = parseInt(number / 10) % 10;
+let number3 = number % 10;
+if (number1===number2 || number2===number3 || number1===number3) {
+console.log('Найдено совпадение цифр');
+} else {
+console.log('Совпадений не найдено');
+}
+
+/*4. Запросить у пользователя год и проверить, високосный он или нет. 
+Високосный год либо кратен 400, либо кратен 4 и при этом не кратен 100.*/
+const leapYear = (year) => (year % 100 === 0 ? year % 400 === 0 : year % 4 === 0);
+console.log(leapYear(2020)); // true
+
+/*6. Написать конвертор валют. Пользователь вводит количество USD, выбирает, 
+в какую валюту хочет перевести EUR, UAN или AZN, и получает в ответ соответствующую сумму.*/
+let USD = 30;
+const EUR = 0.84;
+const UAN = 28.50;
+const AZN = 1.70;
+let result;
+let currency = "UAN";
+switch (currency) {
+  case "EUR": result = USD * EUR;
+    break;
+  case "UAN": result = USD * UAN;
+    break;
+  case "AZN": result = USD * AZN;
+    break;
+  default: result = "not correct currency";
+}
+console.log(result);
+
+/*7. Запросить у пользователя сумму покупки и вывести сумму к оплате со скидкой: от 200 до 300 – скидка будет 3%, 
+от 300 до 500 – 5%, от 500 и выше – 7%.*/
+let sale1 = 0.03;
+let sale2 = 0.05;
+let sale3 = 0.07;
+let amount = 500;
+if (amount >= 200 && amount < 300) {
+  console.log(amount = amount - (amount * sale1))
+} else if (amount >= 300 && amount < 500) {
+  console.log(amount = amount - (amount * sale2))
+} else if (amount >= 500) {
+  console.log(amount = amount - (amount * sale3))
+} else {
+  console.log("unknown")
+}
+
+/*8. Запросить у пользователя длину окружности и периметр квадрата. 
+Определить, может ли такая окружность поместиться в указанный квадрат.*/
+let circleLength = 80;
+let squarePerimetr = 90;
+
+if (circleLength < squarePerimetr) {
+  console.log("Circle will fit in the square")
+} else if (circleLength > squarePerimetr) {
+  console.log("Circle will not fit in the square!")
+} else {
+  console.log("undefinded")
+}
+
+/*9. Задать пользователю 3 вопроса, в каждом вопросе по 3 варианта ответа. За каждый правильный ответ начисляется 2 балла. 
+После вопросов выведите пользователю количество набранных баллов.*/
+let questionToUser1 = +prompt("How much will be 2 * 2: \n6, 4, 8?");
```

Comment:
to get number for string we can use Number(prompt("How much will be 2 * 2: \n6, 4, 8?"))